### ISSUE TITLE
Should show the real key value in some description in the editor.

### ISSUE THREAD

**Issue by andy840119** [karaoke-dev/karaoke#1142]
![image](https://user-images.githubusercontent.com/9100368/155876863-0084b839-aeb0-48ae-8fea-121e5d15e0af.png)
```csharp
protected override Dictionary<LyricEditorMode, EditModeSelectionItem> CreateSelections()
  => new()
  {
      {
          LyricEditorMode.CreateTimeTag,
          new EditModeSelectionItem("Create", "Use keyboard to control caret position, press `N` to create new time-tag and press `D` to delete exist time-tag.")
      },
      {
          LyricEditorMode.RecordTimeTag, new EditModeSelectionItem("Recording", "Press spacing button at the right time to set current time to time-tag.")
      },
      {
          LyricEditorMode.AdjustTimeTag, new EditModeSelectionItem("Adjust", "Drag to adjust time-tag time precisely.")
      }
  };
```
For now, we use fake(or default) key value in here in the description.
But user might be able to change the key they want.
.
So should get real value in here.

**Comment by andy840119** [comment]
Also, what's need to be adjusted:
- `CreateSelections()` should be called in the BDL, not in the constructor.
- Find a way to get the key name.
- Think about is there any need to pass the param in the `CreateSelections`?

**Comment by andy840119** [comment]
Note:
there's no need to consider the 1. and 3. parts if user class instead of text to handler the hotkey name (e.g: `D`)
See the proposal in #1144

**Comment by andy840119** [comment]
Guess should implemented in #1145 also.
